clients/drcachesim/tools/func_view.cpp

```diff
@@ -31,7 +31,7 @@
  */
```

Comment:
> While debugging a malformed trace, I hit multiple crashes in func_view. This update eliminates two causes of these crashes: the indent string now handles negative indent levels by treating them as 0, and functions with no known names are labeled by their function number.

Please update the description with the latest summary.

Reply:
Will do.